pkgs/misc/emulators/stella/default.nix

```
@@ -1,18 +1,22 @@
-{ stdenv, fetchurl, pkgconfig, SDL2 }:
+{ stdenv, fetchFromGitHub, pkgconfig, SDL2 }:
 
 with stdenv.lib;
 stdenv.mkDerivation rec {
 
   pname = "stella";
-  version = "4.6.1";
+  version = "6.1.1";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/project/stella/stella/${version}/${pname}-${version}-src.tar.gz";
-    sha256 = "126jph21b70jlxapzmll8pq36i53lb304hbsiap25160vdqid4n1";
+  src = fetchFromGitHub {
+    owner = "stella-emu";
+    repo = "stella";
+    rev = version;
+    sha256 = "1iwhslrkq887v035j68lhblybww8r792515rp2m5qzmdgnjzsvbb";
   };
 
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ SDL2 ];
+  
+  enableParallelBuild = true;
```

```suggestion
  enableParallelBuilding = true;
```
https://nixos.org/nixpkgs/manual/#ssec-stdenv-attributes